.github/workflows/release-3.yml

```diff
@@ -12,7 +12,9 @@ concurrency:
 jobs:
   publish:
     runs-on: ubuntu-latest
-    environment: npm
+    environment:
+      name: npm
+      url: https://npmjs.com/package/dd-trace
     permissions:
       id-token: write
       contents: write
```

Comment:
Should we just remove the release script for such an outdated release line? I doubt we'll release a new version for v3?